amccarthy9904/ListShare
# Fix missing form, configure project

### Description
- Fixed form issue
- Added a readme with steps on how to setup the project
- Removed files that shouldn't have been committed
- Added a gitignore to avoid committing unwanted files
- Added virtual env to make setup easier

### Testing Suggestions
1. Run ```pip install pipenv``` to install pipenv system wide
2. Run ```pipenv --python 3.6`` to create a virtualenv with django installed (See: Pipfile.lock and Pipfile to understand how that works)
3. Run ```pipenv run python manage.py runserver 8000```  to start your server at localhost:8000
4. Hit ```cntl+c``` to kill the server
4. Verify that everything still runs as expected
5. Run ```git remote add sage https://github.com/sagersmith8/ListShare.git``` to add my remote
6. Run ```git fetch sage``` to download my branch
7. Run ```git checkout some_fixes``` to checkout my branch
8. Run ```pipenv run python manage.py runserver 8000```  to start my server at localhost:8000
9. Visit [localhost:8000](https://localhost:8000) and under Testlist select Edit and you will see that content now shows up

### Other suggestions to make this look more professional
- Look into [Travis CI](https://travis-ci.com/) for continuous integration
- Look into [Pytest](https://docs.pytest.org/en/latest/) for unit tests that you can run on you CI
- Look into [Codecov](https://codecov.io/?gclid=CjwKCAjw1dzkBRBWEiwAROVDLEPVv3NzXp-LtbcavS7P8ZeQ0ZxjjMYDCZ61LiR-rE9QSF6hstqRNhoC314QAvD_BwE) to show code coverage for unit tests
- Look into adding some badges to your readme as they apply
- Bonus points if you are able to get this to publish to Pypi

### Please Review
@amccarthy9904
